### ISSUE TITLE
Escape HTML in markdown 

### ISSUE THREAD

**Issue by kulakowka** [styleguidist/react-styleguidist#284]
I'm want to use markdown in the comments to the source code of my components..

<img width="321" alt="2017-01-12 2 21 36" src="https://cloud.githubusercontent.com/assets/557190/21870531/e3fa269a-d86d-11e6-99e9-453cbb259315.png">

But when I generate documentation, `<code>` tag is processed incorrectly.

<img width="355" alt="2017-01-12 1 59 46" src="https://cloud.githubusercontent.com/assets/557190/21869919/df4c6296-d86a-11e6-9320-379b064791a3.png">
<img width="583" alt="2017-01-12 1 59 57" src="https://cloud.githubusercontent.com/assets/557190/21869918/df4a9434-d86a-11e6-8ac6-95886aaf4ddc.png">

There must be escaped HTML: 

```
<code class="lang-html">
&lt;p&gt;custom html&lt;/p&gt;
</code>
```

But this is not happening.

I think you should escape the HTML here: https://github.com/styleguidist/react-styleguidist/blob/master/src/utils/markdown-to-jsx.js#L426 

But I'm not 100% sure. 🍺 🍺 🍺

**Comment by sapegin** [comment]
It should be fixed in the 5.0.0 beta. Could you please try it?

https://github.com/styleguidist/react-styleguidist/releases/tag/5.0.0%40beta.9

**Comment by sapegin** [comment]
Closing this for now. Feel free to reopen if you still have this issue.
